### ISSUE TITLE
Adding the public dataset `spider` to our git-annex server

### ISSUE THREAD

**Issue by NathanMolinier** [neuropoly/data-management#293]
## Description

To allow easier access to the data and allow the creation of new labels, the public dataset [spider](https://spider.grand-challenge.org/data/) will be pushed to our git-annex server data.

This dataset contains lumbar spine MRI scans obtained from 257 patients "who had experienced Lumbar Back Pain (LBP) in the past":
- 196 T1w sagittal low resolution scans: "ranged from 3.3 x 0.33 x 0.33 mm to 4.8 x 0.90 x 0.90 mm"
- 210 T2w sagittal low resolution scans: "ranged from 3.3 x 0.33 x 0.33 mm to 4.8 x 0.90 x 0.90 mm"
- 41 T2w sagittal high resolution scans: "size of 0.90 x 0.47 x 0.47 mm"

Regarding the labels, two type of labels are currently available:
- Discrete segmentations of multiple structure of the spine (e.g. vertebrae, discs, spinal canal, spinal cord). These segmentations were not changed.
- Discrete labels positioned at the posterior tip of each disc. Since the data is lumbar, the label `25` was chosen as the reference point: this last label corresponds to the last disc before the sacrum structure from superior to inferior direction.

| Discrete segmentation | Discrete discs labels |
| :---: | :---: |
| ![sub-7_T1w_seg_sag_0 5](https://github.com/neuropoly/data-management/assets/68945192/90c4e49f-649e-4e2d-9b60-89286789a5a6) | <img width="240" alt="Screenshot 2024-01-24 at 14 52 52" src="https://github.com/neuropoly/data-management/assets/68945192/169efc28-96ba-4c65-b374-7277464c69b0"> |

Can someone initialize a new git-annex dataset `data.neuro.polymtl.ca:datasets/spider-challenge-2023` and grant me read and write access ?

**Comment by mguaypaq** [comment]
I created the repository and granted you write access, @NathanMolinier. It's initialized with the current dataset template. You'll need to:
1. Clone the repository.
2. Run `git annex init` and `git annex dead here`.
3. Add your files, make your commits, etc.
4. Follow [the upload instructions](https://intranet.neuro.polymtl.ca/data/git-datasets.html#upload).

**Comment by NathanMolinier** [comment]
I pushed the dataset on a branch called `nm/first-commit`. I think the dataset is now ready for review.

**Comment by mguaypaq** [comment]
The dataset looks fine to me:
* The image files are correctly git-annexed.
* The only warnings reported by bids-validator are the usual: INCONSISTENT_SUBJECTS and INCONSISTENT_PARAMETERS. I added a commit with a `.bids-validator-config.json` file to ignore them.

Since we discussed it in person, I'll add a summary here:
Some of the subjects have more than one T1w or T2w image, taken at different resolutions. Judging by [the example in the BIDS docs](https://bids-specification.readthedocs.io/en/stable/appendices/entities.html#acq), it looks like the way to BIDSify this is to use the `acq` entity in the filename. In this case, either `acq-lowresSag` or `acq-highresSag`.

@jcohenadad, did you want to have a look at the branch `nm/first-commit` before I merge into `master`?
